src/test/java/net/openhft/chronicle/bytes/NativeBytesStoreTest.java

```diff
@@ -265,7 +266,7 @@ public void testCopyTo() {
         final Bytes<ByteBuffer> dst = Bytes.elasticByteBuffer();
         try {
             dst.writePosition(src.copyTo(dst));
-            assertEquals(src, dst);
+            assertTrue(BytesInternal.contentEqual(src, dst));
```

Comment:
suggestion -use AssertEquals(,) but first convert both to a string as this gives you a better diff in the IDE if it fails

Reply:
Because we changed how `Bytes::equals` works (it might consider the contents or it might use `==`) then this is the way to go. We might create a custom method (for testing) that asserts content equality and that shows the difference.

Reply:
Ahh. I did not read your comment well enough. Good suggestion.

Reply:
> Ahh. I did not read your comment well enough. Good suggestion.

are you gonna change it again ?